PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -2278,6 +2278,7 @@ void FindReplaceDlg::enableReplaceFunc(bool isEnable)
 	::ShowWindow(::GetDlgItem(_hSelf, IDC_FINDALL_OPENEDFILES), !hideOrShow);
 	::ShowWindow(::GetDlgItem(_hSelf, IDCCOUNTALL),!hideOrShow);
 	::ShowWindow(::GetDlgItem(_hSelf, IDC_FINDALL_CURRENTFILE),!hideOrShow);
+	::ShowWindow(::GetDlgItem(_hSelf, ID_QUICK_FIND), SW_SHOW);

```

Comment:
Is the `SW_SHOW` deliberate or should it be `!hideOrShow` as is in the three lines above it?
When deliberate add a explanatory comment to prevent anyone to interpret as a copy/paste error.